extension/content-script/index.js

```diff
@@ -1,10 +1,10 @@
 import './extension.css';
-import 'codemirror/mode/javascript/javascript';
-import 'codemirror/mode/gfm/gfm';
 import codeMirror from 'codemirror';
 import tabHandler from './tab-handler';
 import onPageNav from './on-page-nav';
 import { removeElement } from './dom';
+var bulk = require('bulk-require');
```

Comment:
Ahhh, forgot about that limitation - I've hit the same thing with `brfs`. Let's just change it to a `const` (only supporting newer versions of Chrome) and I'll merge it in.